pvr.iptvsimple/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.iptvsimple"
-  version="3.9.8"
+  version="3.10.8"
```

Comment:
Apologies. This should be 3.10.0. If we increment the minor number the micro number gets reset.

Reply:
Sure, I'll do that after work